client/src/components/Seat/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import './style.scss';
+
+const Seat = () => {
+  const [selected, setSelected] = useState(false);
+  // TO DO gest variables from props
+  const occupied = false;
+  const seatNr = 1;
+  // TO DO move the handle function to the parent component
+  const handleSeatSelected = (e) => {
```

Comment:
It should also generate new ticket or delete it on click. Perhaps you can also begin to introduce this logic?